packages/create-remix/index.ts

```diff
@@ -667,10 +670,15 @@ async function installDependencies({
   showInstallOutput: boolean;
 }) {
   try {
-    await execa(pkgManager, ["install"], {
-      cwd,
-      stdio: showInstallOutput ? "inherit" : "ignore",
-    });
+    await execa(
+      pkgManager,
+      ["install", ...(pkgManager === "deno" ? ["--no-lock"] : [])],
```

Comment:
the reason `--no-lock` is passed here is that deno v1 uses a different lockfile format to v2. when running `deno install` in deno v1 with `DENO_FUTURE=1` set, it installs using the lockfile format of v2 which will cause errors running other deno commands without also setting `DENO_FUTURE=1` as well. and `deno install` must be ran with `DENO_FUTURE=1` set as deno v1's `deno install` does something entirely different with no good alternative (`deno cache` doesn't install all deps).

there's three solutions to this i can think of:
- wait for deno 2 to release and only support that, currently there are release candidates available
- force `DENO_FUTURE=1` to be used for the entire project (i.e. when running any deno related command and afaik there is no configuration option for this)
- do what is currently done here and don't create a lockfile in this step, let the user do it in their own time (e.g. when a command is ran like `deno task dev` or adding a new dependency to the project - both will automatically create the lockfile)